.github/workflows/js-tests.yml

```diff
@@ -0,0 +1,31 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
```

Comment:
we can just remove the `branches` option

Reply:
Doing this causes duplicate checks on PRs